### ISSUE TITLE
Google thinks yakyak is a new iOS device

### ISSUE THREAD

**Issue by azizLIGHT** [yakyak/yakyak#104]
I signed in after putting in my 2 factor auth code, and Google sent me an email saying "New sign-in from Chrome on iOS." Google shouldn’t be considering yakyak as a new phone. I user other programs and apps with my google account and none of those get recognized as a new phone, let alone as iOS. I'm accessing yikyak 1.1.1 in Ubuntu 14.04 LTS x64 and maybe that has something to do with it?
![screenshot-mail google com 2016-01-06 04-34-03](https://cloud.githubusercontent.com/assets/8013867/12139636/cb09d3fa-b42f-11e5-8f63-c97e5b02249b.png)
![screenshot-security google com 2016-01-06 04-38-17](https://cloud.githubusercontent.com/assets/8013867/12139639/ce19ae62-b42f-11e5-9073-94b8d3d03d90.png)

**Comment by tiamat2012** [comment]
That's because the OATH2 client ID being used is 'borrowed' from IOS and used in this app.

See /Applications/YakYak 2.app/Contents/Resources/app/node_modules/hangupsjs/src/auth.coffee:

```
# this CLIENT_ID and CLIENT_SECRET are whitelisted at google and
# turns up as "iOS device". access can be revoked a this page:
# https://security.google.com/settings/security/permissions
OAUTH2_CLIENT_ID     = '936475272427.apps.googleusercontent.com'
```

The YakYak developers would have to get their own client ID for it to show up differently.

**Comment by azizLIGHT** [comment]
Is that particularly hard or costs money to do? Why clone iOS

**Comment by davibe** [comment]
We just have never done this. Will think about it.

**Comment by mkzero** [comment]
That would be a *really* nice addition. I just did my routine sec checkup and noticed this too and if changing my password hadn't broken my yakyak I never wouldv'e guessed this could be yakyak...

**Comment by algesten** [comment]
the hangups project have a new way of doing auth. we should adopt it in hangupsjs.

**Comment by psouza4** [comment]
Can we re-open this?  It's still an ongoing issue.

**Comment by cbanack** [comment]
Just installed YakYak.   This issue, while certainly not a showstopper, was the only part of the experience that  gave me pause.   If it's an easy fix, would probably be worth it.

**Comment by SmartCatalog** [comment]
I went through a moment of panic. Very disconcerting.

**Comment by skovmand** [comment]
I had the same issue. I was very worried about it for several hours, couldn't figure out what that ios device was and how it got into my account. This should be fixed or be made very explicit.

**Comment by teicher** [comment]
Same thing here, wasted half an hour wondering, revoking access, then reconnecting YakYak.
What to do to get this going ? Putting unfixed issues to "Closed" is very disappointing to everyone.

**Comment by davibe** [comment]
The README clearly states

"Yakyak may show up as iOS Device and Google may alert you that "some iOS Device is trying to use your account". This is normal as yakyak is an unofficial client and it mimics the behaviour of an iOS device in order to establish a communication with Google Hangout APIs."

Also, this issues is clearly open. Only duplicates are closed.

**Comment by Hurricaaane** [comment]
As an UX enhancement, maybe it would help to display a warning inside the yakyak client on first login?

**Comment by psouza4** [comment]
This is not stale, bot!  Unstalify it!  Sadly, it's still an issue.

**Comment by averissimo** [comment]
@psouza4 thanks!! the bot is useful, but for these ongoing issues it should just learn to be quiet :p

**Comment by cmleininger** [comment]
Also just had a similar panic to everyone else in this thread. Please fix this!

**Comment by lparsons** [comment]
:+1: to get this issue resolved...

**Comment by anixon604** [comment]
Yup... this was confusing. I panicked and changed my password. Took me 15 min to figure out it was yak yak

**Comment by ntsagkas** [comment]
bumb
